### ISSUE TITLE
This line should be removed

### ISSUE THREAD

**Issue by gekailin** [cocos2d/cocos2d-x#16559]
https://github.com/cocos2d/cocos2d-x/blob/v3/tools/simulator/frameworks/runtime-src/proj.android/jni/Application.mk#L12

**Comment by vkreal** [comment]
Is that line dead?

**Comment by gekailin** [comment]
No, but in release build I think this macro should not be present.

**Comment by minggo** [comment]
@gekailin i think you are right, why not send a PR for it?
